interfaces/builtin/tpm12.go

```diff
@@ -0,0 +1,109 @@
+package builtin
+
+import (
+	"bytes"
+
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var tpm12PermanentSlotAppArmor = []byte(`
+# Description: for tcsd
+# Usage: common
+
+#include <abstractions/nameservice>
+
+/dev/tpm0 rw,
+`)
+
+var tpm12ConnectedPlugAppArmor = []byte(`
+# Description: for tpm*
+# Usage: common
+
+#include <abstractions/nameservice>
+`)
+
+var tpm12PermanentSlotSecComp = []byte(`
+# Description: for tcsd
```

Comment:
This shouldn't menation tcsd but describe that those are needed to talk to the system TPM chip over /dev/tpm0